apps/python-sdk/firecrawl/firecrawl.py

```diff
@@ -41,7 +73,8 @@ def scrape_url(self, url: str, params: Optional[Dict[str, Any]] = None) -> Any:
         response = requests.post(
             f'{self.api_url}/v0/scrape',
             headers=headers,
-            json=scrape_params
+            json=scrape_params,
+            timeout=(5,10)
```

Comment:
Might not want to hard code the timeout for scrape.

Reply:
We can prob remove that as the function itself already has a default timeout of 15s